WordPress/Classes/ViewRelated/Support/SupportTableViewController.swift

```diff
@@ -542,7 +540,12 @@ extension SupportTableViewController: SupportChatBotCreatedTicketDelegate {
     }
 }
 
-private class ButtonCell: WPTableViewCellDefault {
+fileprivate enum SupportTableLayoutSpacing {
+    static let topBottomPadding: CGFloat = 16
+    static let sidePadding: CGFloat = 20
```

Comment:
These are moved here to fix the left alignment of "Visit our FAQ". Otherwise these 2 cells were using different paddings on the sides.

Before:

<img width=300 src="https://github.com/wordpress-mobile/WordPress-iOS/assets/5032900/6ff972fb-26bc-4083-8c13-a8a23fb0820f">

After:

<img width="300" alt="Screenshot 2024-02-15 at 15 20 21" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/5032900/4451a8cb-f842-4a7a-9484-bd45b2552c08">